t/10-jobs.t

```diff
@@ -42,6 +42,20 @@ my $t      = Test::Mojo->new('OpenQA::WebAPI');
 my $jobs   = $t->app->schema->resultset("Jobs");
 my $users  = $t->app->schema->resultset("Users");
 
+# Allow Devel::Cover to collect stats for background jobs
+sub cover { Devel::Cover::report() if Devel::Cover->can('report') }
+$t->app->minion->on(
+    worker => sub {
+        my ($minion, $worker) = @_;
+        $worker->on(
+            dequeue => sub {
+                my ($worker, $job) = @_;
+                $job->on(cleanup => \&cover) unless $job->info->{notes}{no_cover};
+            });
+    });
```

Comment:
It would be better not to C&P this from the GRU test and instead add it to the test utilities.

Reply:
Yes, I still wanted to do that. Surprise! There were already 20 other duplications :D Fixed in https://github.com/os-autoinst/openQA/pull/3603/commits/7035fbc776f98e92b20ad5c70ebdc64245d161ff